anesthetic/plot.py

```diff
@@ -534,8 +571,10 @@ def kde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
     ncompress = kwargs.pop('ncompress', 1000)
     label = kwargs.pop('label', None)
     zorder = kwargs.pop('zorder', 1)
-    linewidths = kwargs.pop('linewidths', 0.5)
     color = kwargs.pop('color', next(ax._get_lines.prop_cycler)['color'])
+    facecolor = kwargs.pop('facecolor', color)
+    edgecolor = kwargs.pop(
+        'edgecolor', color if facecolor in [None, 'None', 'none'] else 'k')
```

Comment:
Is `edgecolor='k'` our favourite default? Another option would be `edgecolor=color`...

Reply:
Could you provide a picture to show the difference here too?

Reply:
Here a direct comparison of the two cases. Black contours make plots a bit crisper on their own, however it becomes cluttered more quickly when having multiple contours per axes.

Generated with:
```python
ns = NestedSamples(root="anesthetic/tests/example_data/pc")
fig = plt.figure(figsize=(8, 8))
fig, axes = make_2d_axes(['x0', 'x1', 'x2', 'x3', 'x4'], fig=fig)
ns.plot_2d(axes, c='C0', ec='k' , types=dict(lower='kde', diagonal='kde'));
ns.plot_2d(axes, c='C0', ec='C0', types=dict(upper='kde'));
```

![unfilled](https://user-images.githubusercontent.com/18258042/88595217-79f5b000-d05a-11ea-94f0-c13a7e209e10.png)

Reply:
Nice! I really don't know which I prefer, but here are arguments in favour of `'k'`:
- Preserving backward-compatible behaviour
- The other variant will be more useful for large, complicated plots, where code compactness is probably less useful.
I imagine I'll make a lot of use of this regardless, so this is an excellent addition.

Reply:
Ok, I'll keep `'k'` as default, then.